src/sagas/search.js

```diff
@@ -0,0 +1,21 @@
+import {FETCH_SEARCH_GIFS, fetchSearchGifsFail, fetchSearchGifsOk} from "../actions/search";
+import {call, put, takeEvery} from "redux-saga/effects";
+import get from './giphyAPI';
+import {simplifyGIFObject} from "./utils";
+
+function * onFetchSearchGifs (action) {
+  let response;
+  try {
+    response = yield call(get, `search?q=${action.payload}&limit=80`);
+  } catch (error) {
+    yield put(fetchSearchGifsFail());
+  }
+  const searchGifs = response.map(simplifyGIFObject);
```

Comment:
These lines 13 and 14 should be inside the try block. Look at this case: if there's an error you are going to dispatch two actions: fetchSearchGifsFail and fetchSearchGifsOk.